### PR TITLE
feat: Tablet - Comment Lead and Cartoon

### PR DESCRIPTION
**Pull request by nehasri89**
This slice requires two new tiles as it had headline font changes in addition to padding changes
- Tile AH
- Tile AI

### Android Tablet
Screenshot 2019-03-18 at 16 31 43

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-ai/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { View } from "react-native";
+import PropTypes from "prop-types";
+import { getCrop, TileImage, TileLink, withTileTracking } from "../shared";
+import styles from "./styles";
+
+const TileQ = ({ onPress, tile }) => (
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-ai/index.js]: Could we rename it to TileAI, please?

File: packages/edition-slices/src/tiles/tile-ah/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import {
+  getCrop,
+  TileImage,
+  TileLink,
+  TileSummary,
+  withTileTracking
+} from "../shared";
+import styles from "./styles";
+
+class TileP extends Component {
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-ah/index.js]: Could we rename it to TileAH, please?

File: packages/slice-layout/src/templates/commentleadandcartoon/styles.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { editionBreakpoints } from "@times-components/styleguide";
+
+const smallBreakpointStyles = {};
```
Comment (jimhunty) [inline packages/slice-layout/src/templates/commentleadandcartoon/styles.js]: To me this seems unnecessary. There is no need for a style factory here.

Comment (nehasri89) [reply packages/slice-layout/src/templates/commentleadandcartoon/styles.js]: this was kept keeping in mind if there are design changes in future, we can use this, but I can remove it if you think?

File: packages/edition-slices/__tests__/shared-tablet-slices.base.js [hunk 4]
```diff
@@ -87,6 +89,11 @@ const slices = [
     mock: mockSecondaryTwoNoPicAndTwoSlice(),
     name: "secondary two no pic and two slice",
     Slice: SecondaryTwoNoPicAndTwoSlice
+  },
+  {
```
Comment (aronshamash) [inline packages/edition-slices/__tests__/shared-tablet-slices.base.js]: these are mostly in alphabetical order can we add to top ?

File: packages/edition-slices/src/tiles/tile-ah/styles/index.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    color: colours.section.comment,
+    fontFamily: fonts.supporting,
+    fontSize: fontSizes.cardMetaMobile,
+    letterSpacing: 0.6,
+    lineHeight: 12
+  },
+  container: {
+    alignItems: "center",
+    paddingHorizontal: spacing(4),
+    paddingVertical: 16
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/tile-ah/styles/index.js]: `spacing(3)` ?

File: packages/edition-slices/src/tiles/tile-ai/styles/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { spacing } from "@times-components/styleguide";
+
+const styles = {
+  container: {
+    alignItems: "center",
+    paddingBottom: 16,
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/tile-ai/styles/index.js]: `spacing(3)`

File: packages/slice-layout/__tests__/clac.base.js [hunk 7]
```diff
@@ -1,12 +1,13 @@
...
+import { editionBreakpoints } from "@times-components/styleguide";
 import { iterator } from "@times-components/test-utils";
 import createItem from "./utils";
 import { CommentLeadAndCartoon } from "../src/slice-layout";
 
 export default renderComponent => {
   const tests = [
     {
-      name: "two child element",
+      name: "comment lead and cartoon",
```
Comment (aronshamash) [inline packages/slice-layout/__tests__/clac.base.js]: `comment lead and cartoon - small`

File: packages/slice-layout/__tests__/clac.base.js [hunk 8]
```diff
@@ -15,6 +16,20 @@ export default renderComponent => {
           />
         );
 
+        expect(output).toMatchSnapshot();
+      }
+    },
+    {
+      name: "tablet - comment lead and cartoon",
```
Comment (aronshamash) [inline packages/slice-layout/__tests__/clac.base.js]: `comment lead and cartoon - medium`

Comment (aronshamash) [reply packages/slice-layout/__tests__/clac.base.js]: see https://github.com/newsuk/times-components/pull/1812

File: packages/slice-layout/src/templates/commentleadandcartoon/styles.js (hunk 3 above)
Comment (aronshamash) [reply packages/slice-layout/src/templates/commentleadandcartoon/styles.js]: I would remove until we need it.

File: packages/slice-layout/src/templates/commentleadandcartoon/styles.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+const mediumBreakpointStyles = {
+  cartoon: {
+    width: "67%"
+  },
+  container: {
+    flex: 1,
+    flexDirection: "row",
+    paddingRight: 20
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/commentleadandcartoon/styles.js]: `spacing(4)`

File: packages/edition-slices/src/tiles/tile-ah/styles/index.js (hunk 5 above)
Comment (nehasri89) [reply packages/edition-slices/src/tiles/tile-ah/styles/index.js]: zeplin specifically says 16, you think we should round it off ?

Comment (aronshamash) [reply packages/edition-slices/src/tiles/tile-ah/styles/index.js]: yes

**Pull request merged**